src/coreComponents/events/EventManager.cpp

```diff
@@ -54,7 +54,7 @@ EventManager::EventManager( string const & name,
     setDescription( "Start simulation time for the global event loop." );
 
   registerWrapper( viewKeyStruct::maxTimeString(), &m_maxTime ).
-    setApplyDefaultValue( std::numeric_limits< real64 >::max() ).
+    setApplyDefaultValue( 1000 * units::YearSeconds ). // 1000 years
```

Comment:
Let's do 10000 just in case.
```suggestion
    setApplyDefaultValue( 10000 * units::YearSeconds ). // 10000 years
```